webapp/carts/models.py

```diff
@@ -0,0 +1,42 @@
+from django.conf import settings
+from django.db import models
+
+
+User = settings.AUTH_USER_MODEL
+
+
+class Cart(models.Model):
+    #allowed nulls and blanks for future guest checkout
+    user = models.ForeignKey(
+        User,
+        null=True,
+        blank=True)
+
+    book = models.ForeignKey(
+        'book_details.Book',
+        related_name='product')
+
+    total_price = models.DecimalField(
```

Comment:
I don't think that this should be a field.
This is an implicit/derived attribute on the state of the cart.
We'd have to constantly update this as users make changes, which is more cumbersome than it needs to be

Reply:
this is here until the orders field is developed.

Reply:
I'm not sure I see how this is needed; price can always be derived from products.
We currently have no plans of having modifiable prices for items and pricing of an item should be associated to a book independently of what goes on here.